tests/clone_test.go

```diff
@@ -0,0 +1,53 @@
+package tests_test
```

Comment:
I'd like to see 3 new dirs under _tests/_: importer/, cloner/, and dataVolume/. I think it may get messy have all of our e2e tests under a single dir.